client/components/payment-methods-checkboxes/payment-method-checkbox.tsx

```diff
@@ -47,10 +48,10 @@ const getDescription = ( name: string, currency: string ) => {
 const PaymentMethodDescription: React.FC< PaymentMethodProps > = ( {
 	name,
 } ) => {
-	const [ stripeAccountDefaultCurrency ] = useAccountDefaultCurrency();
+	const [ stripeAccountCountry ] = useAccountCountry();
 	const description = getDescription(
 		name,
-		stripeAccountDefaultCurrency as string
+		BNPL_COUNTRY_CURRENCY_MAP[ stripeAccountCountry as string ]
```

Comment:
Below you've introduced `getBNPLCountryCurrencyMap` in different component and it handles the case when account country is not BNPL.
1. Could we extract that method in utils and reuse here?
2. What happens when country is not in `BNPL_COUNTRY_CURRENCY_MAP`?